types/office-js-preview/index.d.ts

```diff
@@ -1696,7 +1696,9 @@ declare namespace Office {
      */
     interface SaveSettingsOptions {
         /**
-         * Indicates whether the setting will be replaced if stale.
+         * **Warning**: This setting is depreciated and should not be used. It has no effect on most platforms and will cause errors when set to false in Excel on the Web.
```

Comment:
Here and in office-js
```suggestion
         * **Warning**: This setting is deprecated and should not be used. It has no effect on most platforms and will cause errors when set to false in Excel on the Web.
```